src/draw/sw/lv_draw_sw_layer.c

```diff
@@ -43,54 +43,56 @@
 struct _lv_draw_layer_ctx_t * lv_draw_sw_layer_create(struct _lv_draw_ctx_t * draw_ctx, lv_draw_layer_ctx_t * layer_ctx,
                                                       lv_draw_layer_flags_t flags)
 {
-    if(LV_COLOR_SCREEN_TRANSP == 0 && (flags & LV_DRAW_LAYER_FLAG_HAS_ALPHA)) {
-        LV_LOG_WARN("Rendering this widget needs LV_COLOR_SCREEN_TRANSP 1");
-        return NULL;
-    }
+    do {
+        if(LV_COLOR_SCREEN_TRANSP == 0 && (flags & LV_DRAW_LAYER_FLAG_HAS_ALPHA)) {
```

Comment:
Can you or someone else explain to me the new? meaning of `LV_COLOR_SCREEN_TRANSP`? I remember in LVGL 8.2, this option was only allowed when the display color depth is also 32 bit (which isn't the case for e.g. with RGB565 displays)

https://github.com/lvgl/lvgl/blob/58b75f1175504fc7a020cd3b89937c52d954a5f8/docs/overview/display.md?plain=1#L65-L73

Reply:
I am not the right guy to answer this question, sorry.

Reply:
Something like transparent screens are required when LVGL needs to render an ARGB layer. For example render an object which has a radius (the rendered image remains transparent on the corners). 

To allow "render to transparent background" related features `LV_COLOR_SCREEN_TRANSP` needs to be enabled. It's really counter intuitive now and I'll will refactor this part in v9.